data_structures/arrays/pairs_with_given_sum.py

```diff
@@ -23,6 +23,19 @@ def pairs_with_sum(arr: list, req_sum: int) -> int:
     return len([1 for a, b in combinations(arr, 2) if a + b == req_sum])
 
 
+# Without using the  itertools Package we can do it too
+def no_of_pairs(arr: list, req_sum: int) -> int:
```

Comment:
```suggestion
def no_of_pairs(arr: list[int], req_sum: int) -> int:
```
Please specify the type of the list's contents